src/ui/routing/zcl_abapgit_services_git.clas.abap

```diff
@@ -246,4 +258,13 @@ CLASS zcl_abapgit_services_git IMPLEMENTATION.
     MESSAGE lv_text TYPE 'S'.
 
   ENDMETHOD.
+  METHOD validate_before_push.
+
+    zcl_abapgit_exit=>get_instance(  )->validate_before_push(
+        is_comment = is_comment
+        io_stage   = io_stage
+        io_repo    = io_repo ).
+
+  ENDMETHOD.
```

Comment:
this indirection is not needed? just call the exit method directly?

Reply:
fixed